### DISCUSSION TITLE
I have a permission to change issue and PR in one organisation, I can do it in Browser, but in app I have an error when changing milestone of PR - Panuchi does not have permission to update the pull request … (I’m owner of this PR in organisation)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#50265] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.104.0 (93753009)
Device: iPhone 11 (iOS 16.3.1)
</details>

**Comment by aconsuegra** [comment]
Hi @Panuchi, thanks for getting in touch! Do you see this error only trying to change a milestone? Or do you see it with other fields like labels, assignees, etc.

**Comment by jsoref** [comment]
Duplicate of https://github.com/community/community/discussions/3970

**Comment by ghost** [comment]
issue that referenced this Pr i can do only
in mobile app, site no

Others can Do only on the website

**Comment by ghost** [comment]
@aconsuegra
